docs/user-guide/release-notes/2.16.x.rst

```diff
@@ -5,14 +5,31 @@ Pulp 2.16 Release Notes
 Pulp 2.16.2
 ===========
 
-This is a security fix for CVE-2018-1090 which leaks security info via the `last_override_config` on
-importers and distributors serialized via the Rest API. See `issue 3521 <https://github.com/pulp/pulp/pull/3513>`_
-for more information. With this release, that field no longer reports the information saved in the
-database. Instead it will always show as::
+This release contains a security fix for CVE-2018-1090 which leaks security info via the
```

Comment:
@dkliban are you planning to cherry pick this back to 2.16-release?

Reply:
yes ... it cherry-picks clean t othere.